client/src/modules/Mentor/pages/Students/index.tsx

```diff
@@ -60,7 +60,7 @@ export function Students() {
                 student.active ? (
                   <Link key="expel" href={routes.getExpelRoute(alias)} legacyBehavior>
                     <Button type="link" icon={<MinusCircleTwoTone twoToneColor="red" />}>
-                      Expel
+                      Modify Assignment
```

Comment:
@dzmitry-varabei for me it sounds too corporate style.  Maybe `Change Status` ?

@aliakseitokarev also I would change icon. Current red stop sign looks dangerous. You can pick fro icon set https://ant.design/components/icon

Reply:
ok, let's use “Change Status”